.github/workflows/sdks.yml

```diff
@@ -148,9 +148,9 @@ jobs:
           set -x
 
           if ${{ startsWith(matrix.os, 'macos') }}; then
-            brew install ninja patchelf
+            brew install ninja patchelf tree
           elif ${{ startsWith(matrix.os, 'ubuntu') }}; then
-            sudo apt install ninja-build
+            sudo apt install ninja-build tree
```

Comment:
These two `tree` additions are unnecessary, as you're running it in a subsequent job that uses a fresh image anyway.